api/mesh/v1alpha1/circuit_breaker_helpers.go

```diff
@@ -0,0 +1,26 @@
+package v1alpha1
+
+import (
+	"github.com/golang/protobuf/proto"
+
+	util_proto "github.com/kumahq/kuma/pkg/util/proto"
+)
+
+func (m *CircuitBreaker) UnmarshalJSON(data []byte) error {
+	return util_proto.FromJSON(data, m)
+}
+
+func (m *CircuitBreaker) MarshalJSON() ([]byte, error) {
+	return util_proto.ToJSON(m)
+}
+func (t *CircuitBreaker) DeepCopyInto(out *CircuitBreaker) {
+	proto.Merge(out, t)
```

Comment:
`proto.Merge` takes `Message` parameters, but here you are passing the full CRD object? Do you need to deep copy the meta and other fields before copying the protobuf?

Reply:
In this scope, `CircuitBreaker` is the `Message`